.travis.yml

```diff
@@ -73,6 +73,7 @@ script:
 # using "&&" instead of several "-" means that integrationTest does not run if test fails,
 # and Docker test does not run if integration test fails, which makes PR failure easier to understand.
 # @see https://docs.travis-ci.com/user/job-lifecycle/#customizing-the-build-phase
+# NOTE: Sleep after docker-compose increased to 60 seconds as often Travis would fail to get Docker up in 30 seconds
```

Comment:
@ptuomola just noticed while merging this that you haven't actually changed the `sleep 30s` so this comment is perhaps a left over? I'll merge this PR anyway, you can always clean it up later, if you like.